### PR TITLE
README: fix formatting & links

### PR DESCRIPTION
**Pull request by Mikaela**
[CI SKIP]

### CODE REVIEW

**Comment by Mikaela** [comment]
* https://github.com/znc/znc/pull/989/files?short_path=04c6e90#diff-04c6e90faac2675aa89e2176d2eec7d8

File: README.md [hunk 1]
```diff
@@ -41,114 +41,113 @@ cyrusauth:
...
 For setting up a configuration file in `~/.znc` you can simply do
-	znc --makeconf
-or
-	./znc --makeconf
-for in-place execution.
+`znc --makeconf` or `./znc --makeconf` for in-place execution.
 
-If you are using SSL you should do
-	znc --makepem
+If you are using SSL you should do `znc --makepem`
```
Comment (Mikaela) [inline README.md]: Doesn't makeconf do this?

File: README.md [hunk 2]
```diff
@@ -41,114 +41,113 @@ cyrusauth:
...
-exception are modules and module data, which are saved in `<prefix>/lib/znc`
-and `<prefix>/share/znc`, and the znc binary itself.
+exception are modules and module data, which are saved in
+`<prefix>/lib/znc` and `<prefix>/share/znc`, and the znc binary itself.
 More modules (e.g. if you install some later) can be saved in
 `<data dir>/modules` (-> `~/.znc/modules`).
 
 In the datadir are only two files:
 
 - `znc.pid` - The pid of the currently running ZNC instance.
```
Comment (Mikaela) [inline README.md]: This isn't even enabled by default so is it supposed to be here?

File: README.md [hunk 3]
```diff
@@ -41,114 +41,113 @@ cyrusauth:
...
 ## Further infos
 
-Please visit http://znc.in/ or #znc on freenode if you still have questions.
+Please visit http://znc.in/ or
+[#znc on freenode](ircs://irc.freenode.net:6697/#znc) if you still have
+questions.
 
 You can get the latest development version with git:
-	git clone git://github.com/znc/znc.git
+`git clone https://github.com/znc/znc.git`
```
Comment (Mikaela) [inline README.md]: git isn't encrypted and I think GitHub wishes people to start moving to https as they are hiding the git clone links.

File: README.md [hunk 4]
```diff
@@ -41,114 +41,113 @@ cyrusauth:
...
-	./configure --help
-if you want to get a list of options, though the defaults should be suiting
-most needs. After you compiled it with make (or gmake if make doesn't work) you
-can install it with
-	make install
-though you don't need to as ZNC supports in-place execution.
+You can use `./configure --help` if you want to get a list of options,
+though the defaults should be suiting most needs. After you compiled it
+with make (or gmake if make doesn't work) you can install it with
+`make install` though you don't need to as ZNC supports in-place execution.
```
Comment (DarthGandalf) [inline README.md]: Let's remove the though.... part. It kinda supports such execution, but has multiple flaws.

File: README.md (hunk 1 above)
Comment (DarthGandalf) [inline README.md]: It does.
--makepem is useful only if user has old config without znc.pem for any reason:
* User used --makeconf with ancient version, and didn't enable any SSL listener, but now wants to add one
* ZNC was compiled without SSL support
* The cert expired

File: README.md (hunk 4 above)
Comment (Mikaela) [inline README.md]: The second though, I assume. Done.

File: README.md (hunk 2 above)
Comment (DarthGandalf) [inline README.md]: Probably not...

File: README.md (hunk 1 above)
Comment (Mikaela) [inline README.md]: Should it be removed from here then?

File: README.md [hunk 5]
```diff
@@ -41,114 +41,113 @@ cyrusauth:
...
+items can be found on the
+[Configuration](http://wiki.znc.in/Configuration)-Page.
+**Warning: better not to edit config, while ZNC is running.**
 
-To rehash the config file, you can send ZNC SIGHUP via
-	pkill -SIGHUP znc
-or you can login to ZNC and use
-	/msg *status rehash
+To rehash the config file, you can send ZNC SIGHUP via `pkill -SIGHUP znc`
+or you can login to ZNC and use `/msg *status rehash`
```
Comment (DarthGandalf) [inline README.md]: Let's remove this paragraph altogether

File: README.md (hunk 2 above)
Comment (Mikaela) [inline README.md]: Removed.

File: README.md (hunk 5 above)
Comment (Mikaela) [inline README.md]: Removed. The saving part stays untouched as you said nothing about it and it's not the same paragraph.

File: README.md (hunk 1 above)
Comment (DarthGandalf) [inline README.md]: I don't have strong opinion on this.

File: README.md (hunk 3 above)
Comment (KindOne-) [inline README.md]: Why not  add `--recursive` like on http://wiki.znc.in/Git ?

Comment (Mikaela) [inline README.md]: I forgot, thanks for the reminder :)

Comment (Mikaela) [inline README.md]: Fixed and also changed wiki to use https instead of git.

File: README.md (hunk 1 above)
Comment (Mikaela) [inline README.md]: I am not touching it then.

File: README.md [hunk 6]
```diff
@@ -41,114 +41,109 @@ cyrusauth:
...
-items can be found on the [Configuration](http://wiki.znc.in/Configuration)-Page.
-Warning: better not to edit config, while ZNC is running.
-
-To rehash the config file, you can send ZNC SIGHUP via
-	pkill -SIGHUP znc
-or you can login to ZNC and use
-	/msg *status rehash
+items can be found on the
+[Configuration](http://wiki.znc.in/Configuration)-Page.
+**Warning: better not to edit config, while ZNC is running.**
```
Comment (DarthGandalf) [inline README.md]: With such warning, it's good to provide alternative: webadmin/controlpanel

Comment (Mikaela) [inline README.md]: > **Warning: better not to edit config, while ZNC is running.** Use  the
[webadmin] and [controlpanel] modules instead.

[webadmin]:http://wiki.znc.in/Webadmin
[controlpanel]:http://wiki.znc.in/Controlpanel

Done

**Pull request merged**